Stratis.Bitcoin/Features/BlockStore/LoopSteps/DownloadBlockStep.cs

```diff
@@ -13,27 +13,28 @@ namespace Stratis.Bitcoin.Features.BlockStore.LoopSteps
     ///<list>
     /// There are two operations:
     ///     <item>1: <see cref="BlockStoreInnerStepFindBlocks"/> to ask the block puller to download the blocks.</item>
-    ///     <item>2: <see cref="BlockStoreInnerStepDownloadBlocks"/> to persist the blocks in a batch to the <see cref="BlockRepository"/>.</item>
+    ///     <item>2: <see cref="BlockStoreInnerStepReadBlocks"/> to persist the blocks in a batch to the <see cref="BlockRepository"/>.</item>
     /// </list>
     /// </para> 
     /// <para>
-    /// After a "Stop" condition is found the FindBlocksTask will be removed from 
-    /// the <see cref="BlockStoreInnerStepContext.InnerSteps"/> and only the 
-    /// <see cref="BlockStoreInnerStepDownloadBlocks"/> will continue to execute until the <see cref="BlockStoreInnerStepContext.DownloadStack"/> is empty.
+    /// After a "Stop" condition is found the AskBlocksTask will be removed from
+    /// <see cref="BlockStoreInnerStepContext.InnerSteps"/> and only the 
+    /// <see cref="BlockStoreInnerStepReadBlocks"/> task will continue to execute 
+    /// until the DownloadStack is empty.
     /// </para>   
     /// </summary>
     internal sealed class DownloadBlockStep : BlockStoreLoopStep
     {
-        /// <summary>Instance logger.</summary>
-        private readonly ILogger logger;
-
         /// <summary>Provider of time functions.</summary>
         private readonly IDateTimeProvider dateTimeProvider;
 
+        /// <summary>Instance logger.</summary>
+        private readonly ILogger logger;
+
         internal DownloadBlockStep(BlockStoreLoop blockStoreLoop, ILoggerFactory loggerFactory, IDateTimeProvider dateTimeProvider)
             : base(blockStoreLoop, loggerFactory)
         {
-            this.logger = loggerFactory.CreateLogger(this.GetType().FullName);
+            this.logger = loggerFactory.CreateLogger(GetType().FullName);
```

Comment:
ctrl+shift+f reveals that only BS code is using GetType() without `this.` all other code follows the code style rule for `this.`, let's change all these to follow the code style